packages/frontend/src/pages/Templates/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Center, chakra, Flex, Image, Text } from '@chakra-ui/react'
+import { Badge, Link } from '@opengovsg/design-system-react'
+
+import templatesPreviewImg from '@/assets/templates-preview.svg'
+import Container from '@/components/Container'
+import MarkdownRenderer from '@/components/MarkdownRenderer'
+import * as URLS from '@/config/urls'
+
+export default function Templates(): JSX.Element {
+  return (
+    <Container w="50%">
```

Comment:
page will be squashed in mobile view if u hardcode the width as such.

Reply:
ok ill vary it based on the screensizes for now, same thing this will change in the next few PRs